external/mqtt/src/main/scala/org/apache/spark/streaming/mqtt/MQTTUtils.scala

```diff
@@ -74,3 +75,24 @@ object MQTTUtils {
     createStream(jssc.ssc, brokerUrl, topic, storageLevel)
   }
 }
+
+/**
+ * This is a helper class that wraps the methods in MQTTUtils into more Python-friendly class and
+ * function so that it can be easily instantiated and called from Python's MQTTUtils.
+ */
+private class MQTTUtilsPythonHelper {
+
+  def createStream(
+      jssc: JavaStreamingContext,
+      brokerUrl: String,
+      topic: String,
+      storageLevel: StorageLevel
+    ): JavaDStream[Array[Byte]] = {
+    val dstream = MQTTUtils.createStream(jssc, brokerUrl, topic, storageLevel)
+    dstream.map(new Function[String, Array[Byte]] {
+      override def call(data: String): Array[Byte] = {
+        data.getBytes("UTF-8")
```

Comment:
You don't need to convert to bytes. PythonRDD will do it. You can pass a Sting DStream and use `UTF8Deserializer` in Python directly.